custom/app-setup/kafka/ansible_kafka_cleanup.yml

```
@@ -0,0 +1,41 @@
+- name: "[CleanUp]"
+  hosts: localhost
+  vars:
+         ansible_python_interpreter: /usr/bin/python3
+
+  tasks:
+          - name: "[CleanUp]: Removing  the deployments of prometheus"
+            community.kubernetes.k8s:
+                    state: absent
+                    name: prometheus-operator
+                    kind: Deployment
+                    namespace: monitoring
+
+          - name: "[CleanUp: Removing Grafana deployment"
```

```suggestion
          - name: "[CleanUp]: Removing Grafana deployment"
```
Check for other tasks as well